test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/JsonLightMetadataLevelTests.cs

```diff
@@ -29,56 +29,56 @@ public JsonLightMetadataLevelTests()
         public void ODataMinimalMetadataShouldCreateMinimalMetadataLevel()
         {
             this.parameterList.Add(new KeyValuePair<string, string>("odata.metadata", "minimal"));
-            Assert.IsType<JsonMinimalMetadataLevel>(JsonLightMetadataLevel.Create(this.applicationJsonMediaType, MetadataDocumentUri, Model, /*writingResponse*/ true));
+            Assert.IsType<JsonMinimalMetadataLevel>(JsonLightMetadataLevel.Create(this.applicationJsonMediaType, MetadataDocumentUri, /* alwaysAddTypeAnnotationsForDerivedTypes */ false, Model, /*writingResponse*/ true));

```

Comment:
Maybe add an "Create" overload without alwaysAddTypeAnnotationsForDerivedTypes parameter and call "Create" with alwaysAddTypeAnnotationsForDerivedTypes using "false".

So, you don't need to change these test cases.

Reply:
`Create` overload added in the 3rd commit to this PR.

Reply:
why don't call the original "Create" without "alwaysAddTypeAnnotationsForDerivedTypes" parameter?

---
In reply to: [539727657](https://github.com/OData/odata.net/pull/1946#discussion_r539727657) [](ancestors = 539727657)